src/main/resources/db/migration/V6__create_user_table.sql

```diff
@@ -0,0 +1,5 @@
+create table `User` (
+`username` varchar(100) not null,
+`password` varchar(100) not null,
```

Comment:
do we need to store any hashes for encrypting passwords?

Reply:
Passwords are stored as hashed strings in the database which is what the system compares to when a user logs in. when entering the password on front end the system hashes their password and compares it to the hashed password stored in the database